include/cutlass/gemm/collective/xe_mma.hpp

```diff
@@ -98,6 +98,9 @@ struct CollectiveMma<
   using TransformB = TransformB_;
   using ArchTag = typename DispatchPolicy::ArchTag;
 
+  static constexpr bool a_row_major = std::is_same_v<cutlass::detail::StrideToLayoutTagA_t<StrideA>, cutlass::layout::RowMajor>;
+  static constexpr bool b_row_major = std::is_same_v<cutlass::detail::StrideToLayoutTagB_t<StrideB>, cutlass::layout::RowMajor>;;
+
```

Comment:
As discussed last week, we want to move all the logic related to data layout closer to the copy builtin function, ideally within the copy trait definition. `XE_mma` and all the other Cutlass components will work with (M, K, L) or (N, K, L) as the shape, and the stride will indicate the leading dimension. Whether the actual data layout is row-major, column-major, or any other form, Cutlass components will always operate with M-major, N-major, and K-major layouts. The copy trait should be able to interpret that and translate it to the layout that the builtin understands.